docs/modules/ROOT/pages/troubleshooting/index.adoc

```diff
@@ -44,4 +44,110 @@ If you see this error message, it means that the pipeline run has exceeded the d
 spec:
   # ...
   timeout: 1h30m0s  # increase accordingly
-----
\ No newline at end of file
+----
+
+== Failed to push or pull image
+
+Failing to authenticate with a container registry can be hard to debug. To be able to troubleshoot
+effectively, one needs to understand how registry authentication works in Tekton. For the full
+details, see the Tekton link:https://tekton.dev/docs/pipelines/auth/[Authentication] docs. In short:
+
+. Tekton uses a ServiceAccount (named `appstudio-pipeline` by default in Konflux) to run your Pipeline.
```

Comment:
Does Tekton use this SA or does {ProductName}?

Reply:
Both I guess? Tekton uses the SA, Konflux calls it `appstudio-pipeline`